doc/sphinx/Pacemaker_Remote/options.rst

```diff
@@ -90,7 +90,7 @@ Example defining a remote node with the hostname **remote1**:
 
 .. code-block:: none
 
-    # pcs resource create remote1 remote
+    # pcs cluster node add-remote remote1
```

Comment:
A comment about Line 99:

The output of the updated version of the remote node definition command looks like this:
[root@pcmk-1]# pcs cluster node add-remote server=192.168.122.102 port=8938
Error: Host 'server=192.168.122.102' is not known to pcs, try to authenticate the host using 'pcs host auth server=192.168.122.102' command, use --skip-offline to override
No addresses specified for host 'server=192.168.122.102', using 'server=192.168.122.102'
Error: Errors have occurred, therefore pcs is unable to continue

Using 'pcs host auth server=192.168.122.102' as suggested by the output results in this:
[root@pcmk-1]# pcs host auth server=192.168.122.102
Error: Invalid character '=' in host name 'server=192.168.122.102'

This command shows that the provided IP address is authorized, however:
[root@pcmk-1]# pcs host auth 192.168.122.102
Username: hacluster
Password: 
192.168.122.102: Authorized